schema/README.md

```diff
@@ -0,0 +1,30 @@
+# Metadata Schema
+
+This directory contains all predefined metadata types in the form of [JSON schema](https://json-schema.org). We expect every piece of metadata to have the following fields:
+
+- `id`of type _string_. Unique identifier assigned by this metadata service.
```

Comment:
Do we want to use UUID here?

Reply:
That's a good question.

UUID implies this field is 128-bit int. I want to let the implementation decide what kind of ID is  used. (e.g. it can be an incremental integer in mysql)